src/SourceTerm/PointSource.cpp

```diff
@@ -98,6 +101,12 @@ void seissol::sourceterm::transformMomentTensor(real const i_localMomentTensor[3
   for (unsigned m = 6; m < NUMBER_OF_QUANTITIES; ++m) {
     o_momentTensor[m] = 0.0;
   }
+
+  for (unsigned j = 0; j < 3; ++j) {
+    for (unsigned k = 0; k < 3; ++k) {
+        o_velocityComponent[k] += R[k][j] * i_localVelocityComponent[j];
+    }
+  }
 }
```

Comment:
Why don't you just add velocityComponent to o_momentTensor directly? It should be large enough and you only need to change the name of o_momentTensor to something more sensible.